proto/agent.proto

```diff
@@ -295,6 +297,8 @@ message ContainerStat {
 	uint32 key = 17;
 	int64 started = 18;
 	bytes byteKey = 19;
+	float threadCount = 20;
```

Comment:
I wonder why these are float types?

Reply:
Indeed, they should be `uint64`, as the source type is, by bad 🤦‍♂️